docs/sources-asciidoc/src/main/asciidoc/uml/replication-seq-diagram.plantuml

```diff
@@ -0,0 +1,32 @@
+@startuml
```

Comment:
this is what i meant, this files needs to be saved as it is. THe previous png file shouldnt be part of the repository.